packages/data/src/plugins/controls/index.js

```diff
@@ -26,5 +27,14 @@ export default function( registry ) {
 
 			return store;
 		},
+
+		fulfill( reducerKey ) {
```

Comment:
To me, it seems more sensible and easy-to-extend to accept as arguments the full thing of the thing we want to fulfill: with selector name and arguments.

Reply:
The idea was to mimic the "select" and "dispatch" APIs but I hesitated a lot. I don't mind changing, what do you think?

Reply:
On a second thought, this doesn't really make sense as a public API aside for plugins, which means we don't need to mimic the select/dispatch behavior. I'm going to update.